prow/config.yaml

```diff
@@ -12971,7 +12971,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- cron: '1 1 2 6 *' # Run at 01:01 UTC (03:01 CEST) on 2nd June
+- cron: '0 21 * * 6' # Run at 21:00 UTC (23:00 CEST) on saturday
```

Comment:
Actually, I just realized this will overlap with our gce-scale-correctness on saturday (https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-scale-correctness).

Could you instead overlap this job with 'gce-large-performance'? (because that also uses just 2k nodes, so we should have space for this one). Plus I'd expect they run for ~similar amount of time.